pandas/tests/indexes/datetimes/test_indexing.py

```diff
@@ -773,3 +773,14 @@ def test_get_loc_nat(self):
         # GH#20464
         index = DatetimeIndex(["1/3/2000", "NaT"])
         assert index.get_loc(pd.NaT) == 1
+
```

Comment:
maybe better to parmateize over the accepted nulls (do we have a fixture for date time nulls?) and then have an errors version

Reply:
ill put that on my list of follow-ups, we're about 2/3 of the way towards having get_value be internally consistent.

Thoughts on the Series.at outlier?

Reply:
what’s the Series.at outlier?

Reply:
> ill put that on my list of follow-ups, we're about 2/3 of the way towards having get_value be internally consistent.
> 
sure

Reply:
> what’s the Series.at outlier?

in the test here `ser.loc[4]` behaves like `ser.loc[4.0]`, while `ser.at[4]` raises